otx/mpa/seg/incremental/stage.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import numpy as np
+from mmcv import ConfigDict
+
+from otx.mpa.seg.stage import SegStage
+from otx.mpa.utils.config_utils import update_or_add_custom_hook
+from otx.mpa.utils.logger import get_logger
+
+logger = get_logger()
+
+
+class IncrSegStage(SegStage):
```

Comment:
Your comments are totally right. I agree that we don't need to keep stage as long as we only maintain otx now and it has to be disappear in the future. But I'd like to let you the specific design for the runners are not decided yet because we are in the middle of refactoring stage to get to the otxv2 design. I wanted to remove the current unclear inheritance which is very different from other tasks (look at det/cls, they all different and confusing.) We discussed to keep the same folder structure and I'd say that it has to be, whenever. And this clarity and structured stages will surely help refactor for the final design for the future, since this PR includes the separation of configuration with respect to train type. As conclusion, this is not a PR for refactoring whole runners design and I hope we can have some discussion soon for how to merge runners and stages because it is very huge task which requires to have the very different structure compared to the current.